bcos-tool/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ project(bcos-tool VERSION ${VERSION})
 file(GLOB_RECURSE SRCS *.cpp)
 add_library(${TOOL_TARGET} ${SRCS})
 target_compile_options(${TOOL_TARGET} PRIVATE -Wno-error -Wno-unused-variable)
-target_link_libraries(${TOOL_TARGET} PUBLIC ${UTILITIES_TARGET})
+target_link_libraries(${TOOL_TARGET} PUBLIC ${UTILITIES_TARGET} PUBLIC ${SECURITY_TARGET})
```

Comment:
`bcos-tool`should  not depends to `bcos-security`, you should access keyCenter when `EncryptFile` init.